### PR TITLE
Simplify AllAllowed

### PR DESCRIPTION
**Pull request by janisz**
## Description

Since builting authenticator does not return error we don't need it in AllAllowed method.

## Testing Performed

CI

### CODE REVIEW

**Comment by janisz** [comment]
Current dependencies on/for this PR:
* master
  * **PR #2726** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/2726" target="_blank">Graphite</a> 
    * **PR #2722** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/2722" target="_blank">Graphite</a> 
      * **PR #2750** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/2750" target="_blank">Graphite</a> 
        * **PR #2751** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/2751" target="_blank">Graphite</a> 
          * **PR #2752** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/2752" target="_blank">Graphite</a> 
            * **PR #2774** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/2774" target="_blank">Graphite</a> 
              * **PR #2775** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/2775" target="_blank">Graphite</a> 
                * **PR #2776** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/2776" target="_blank">Graphite</a> 
                  * **PR #2777** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/2777" target="_blank">Graphite</a> 
                    * **PR #2778** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/2778" target="_blank">Graphite</a>  👈
                      * **PR #2779** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/2779" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/2778?utm_source=stack-comment).

File: pkg/sac/scope_checker.go [hunk 1]
```diff
@@ -72,26 +63,16 @@ func (c scopeChecker) Allowed(subScopeKeys ...ScopeKey) (bool, error) {
 	return curr.Allowed(), nil
 }
 
-// AnyAllowed checks if access to any of the given subscopes is allowed.
-func (c scopeChecker) AnyAllowed(ctx context.Context, subScopeKeyss [][]ScopeKey) (bool, error) {
```
Comment (rhybrillou) [inline pkg/sac/scope_checker.go:76]: nit: this belongs to PR 2776

**Review by rhybrillou — APPROVED** [review]

**Comment by janisz** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/2778?utm_source=stack-comment).


Job Reference: KaXF6lRV7xj8fvvZbgOl

**Pull request merged**